provisioning_templates/provision/kickstart_default.erb

```diff
@@ -75,8 +75,13 @@ liveimg --url=<%= liveimg_url %> <%= proxy_string %>
 <% else %>
 <%= @mediapath %><%= proxy_string %>
 <% @additional_media.each do |medium| -%>
+  <% if rhel_compatible && @host.operatingsystem.name.downcase.include?("centos") && os_major >= 8 && medium[:url] && medium[:url].include?("AppStream") -%>
```

Comment:
Is it normal to have the indentation like this?

Reply:
I de-indented it.